docs/content/CONTENT.md

```diff
@@ -263,3 +266,13 @@ Each content should be formatted like below
   - type: story
     id: story-id
 ```
+
+**asLink**
+
+If the story doesn't hold any contents but the link to outside, use `asLink` property to indicate it. Note that `id` 
+of the story that has `asLink` won't generate the url for the story.
```

Comment:
might be better to say `but links externally` instead

Reply:
I wonder that gives a wrong impression that `id` of the story works as a link?  `id` doesn't really do anything (at least for users, but still needed internally) when the story has `asLink`